src/Router.tsx

```diff
@@ -123,10 +112,15 @@ const FindNavigator = createStackNavigator(
 const MoreNavigator = createStackNavigator(
   {
     [PV.RouteNames.MoreScreen]: MoreScreen,
+    [PV.RouteNames.DownloadsScreen]: DownloadsScreen,
+    [PV.RouteNames.MyProfileScreen]: MyProfileScreen,
+    [PV.RouteNames.PlaylistScreen]: PlaylistScreen,
+    [PV.RouteNames.PlaylistsScreen]: PlaylistsScreen,
+    [PV.RouteNames.ProfileScreen]: ProfileScreen,
```

Comment:
ProfileScreen and MyProfileScreen? Do you think we could maybe have one screen for both somehow?

Reply:
Probably we could...I partly opted for the My Profile screen to be separate because it can only be viewed by the authenticated user, whereas Profile is for other users. Since My Profile has more sensitive operations I thought maybe it'd be good to separate it...but it's not DRY, so maybe that's not a great idea.